src/navigation/createDismissableStackNavigator.tsx

```diff
@@ -32,10 +32,10 @@ function DismissableStackRouter(
     ...router,
     actionCreators: {
       ...router.actionCreators,
-      dismiss(params?: object) {
+      dismiss(params?: object & {screen: string}) {
         return {
           type: 'NAVIGATE',
-          payload: {name: options.dismissToScreen, params},
+          payload: {name: params?.screen || options.dismissToScreen, params},
```

Comment:
Bit unsure about this change. If you want to navigate to a specific screen, you should use `.navigate()` instead of `.dismiss()`

Reply:
I could not get `navigate()` to work out-of-the-box, as only the screens from the current stack and root stack were available, not the sub screens of the tab navigator for tickets. I can look into what the possibilities are. Maybe navigating to the TicketsScreen, and adding a parameter specifying which tab to open?

Reply:
Added fix for this now.